terraform/aws/addons/keycloak.tf

```diff
@@ -0,0 +1,57 @@
+# Deploy keycloak-operator and keycloak itself
+locals {
+  cluster_issuer     = "letsencrypt-prod"
```

Comment:
[p4] Why suffix `-prod` exist?

Reply:
Its common name for letsencrypt provisioners: letsencrypt-stage - for testing purpose and prod - for normal usage